### PR TITLE
Merge the latest basicnet

### PR DESCRIPTION
**Pull request by ericxtang**
Merging the latest version of `basicnet` into the node.  This adds retry logic.

Commit URL: https://github.com/livepeer/go-livepeer-basicnet/commit/14ac37ea7b14ed1cd001c560778b8c6264e5af19

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 *.dll
 *.so
 *.dylib
+*.vscode
```
Comment (j0sh) [inline .gitignore:6]: This change does not seem to actually be anywhere in basicnet. Can you add it to the ping PR and remove it from here?

For future reference, I usually sync via something like this (actual command untested)

`cd $GOPATH/src/github/livepeer/go-livepeer-basicnet && git archive HEAD | tar -C $GOPATH/src/github/livepeer/go-livepeer/vendor/github/livepeer/go-livepeer-basicnet -xf -`

Then sanity check via `git status`, `git diff` etc. That should prevent extraneous changes from creeping in the commit.

**Review by j0sh — CHANGES_REQUESTED** [review]
Can you specify which basicnet git sha this is coming from in the commit log?

File: .gitignore (hunk 1 above)
Comment (ericxtang) [reply .gitignore:6]: Yeah - this is added for the `go-livepeer` repo instead of the `go-livepeer-basicnet` repo.

**Review by j0sh — APPROVED** [review]

**Comment by j0sh** [comment]
🚢

**Pull request merged**